docs(transformer): improve doc comment formatting

overlookmotel opened the pull request:
Nit. Alter formatting of doc comment. Produces same HTML docs, but is easier to read in the Rust code, and doesn't confuse clippy.

camc314 reviewed (approved):
nice, this actually make vscode format it correctly  in the tooltip as well

overlookmotel committed:
docs(transformer): improve doc comment formatting (#12899)

Nit. Alter formatting of doc comment. Produces same HTML docs, but is easier to read in the Rust code, and doesn't confuse clippy.
